assets/src/scripts/outputs-viewer/components/PrepareButton/PrepareButton.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { useMutation } from "react-query";
+import useFileList from "../../hooks/use-file-list";
+import handleErrors from "../../utils/fetch-handle-errors";
+
+function PrepareButton({ csrfToken, filesUrl, prepareUrl }) {
+  const { data: fileList } = useFileList({ apiUrl: filesUrl });
+
+  const mutation = useMutation(
+    ({ fileIds }) =>
+      fetch(prepareUrl, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+          "X-CSRFToken": csrfToken,
+        },
+        body: JSON.stringify({ file_ids: fileIds }),
+      })
+        .then(handleErrors)
+        .then((response) => response.json()),
+    {
+      mutationKey: "PUBLISH_RELEASE",
+      onSuccess: (data) => {
+        // redirect to URL returned from the API
+        window.location.href = data.url;
+      },
+      onError: (error) => {
+        // eslint-disable-next-line no-console
+        console.error("Error", { error });
```

Comment:
Not for this PR, but we will need to consider how we display error states to users.